packages/moonstone/VideoPlayer/VideoPlayer.js

```diff
@@ -955,6 +924,16 @@ const VideoPlayerBase = class extends React.Component {
 		return true;
 	}
 
+	handleLoadStart = () => {
+		this.firstPlayReadFlag = true;
```

Comment:
I'm not sure if we want to wait until `loadstart` event is fired to reset states. Can we reset when `source` prop updates?

Reply:
I wasn't able to test the a11y features because my board died. I guess the question is how it should behave for a video that hasn't loaded yet (either because it's in progress or failed). I'm not sure what the right answer for that is but I'll investigate.

Reply:
I just realized that `firstPlayReadFlag` has been working incorrectly for a very long time. Even with this fix, the first play button click or play remote button click wouldn't read out "play" as it should. I'll file a separate bug for this.